### PR TITLE
adding files for idr0018

### PR DESCRIPTION
**Pull request by eleanorwilliams**

### CODE REVIEW

**Comment by joshmoore** [comment]
Especially if the file names have spaces, could we move to a TSV (incl. file-ending for proper detection)?

**Comment by joshmoore** [comment]
Also, these use /nfs/bioimage paths. I'll need /uod/idr ones for importing. Can discuss this AM.

**Comment by eleanorwilliams** [comment]
@joshmoore.  I thought we said that it was ok if the data was at EBI only.  I can start the rsync to Dundee today.  

So you want the files with a .tsv file ending rather than .txt?

**Comment by eleanorwilliams** [comment]
@joshmoore.  Ready for testing again when the image files have been copied to Dundee.

File: idr0018-neff-histopathology/experimentA/idr0018-experimentA-bulk.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+include: "../../bulk.yml"
+path: "idr0018-experimentA-filePaths.txt"
```
Comment (joshmoore) [inline idr0018-neff-histopathology/experimentA/idr0018-experimentA-bulk.yml]: this will need to also be updated to "tsv"

**Comment by eleanorwilliams** [comment]
@joshmoore. Should be ready for testing now.  bulk.yml now points to .tsv file.

**Comment by joshmoore** [comment]
Going in now.

**Comment by joshmoore** [comment]
All imported.
bulk annotation complains about some missing files:

```
ERROR:omero.util.populate_metadata:Pipox-14-291-skin - 2015-06-24 04.14.36.ndpi not found in image names
```

**Comment by joshmoore** [comment]
```
[jamoore@minke experimentA]$ omero metadata populate Project:101 --file idr0018-experimentA-annotation.csv --dry-run  2>&1 | grep "not found in image names" | wc
    231    2076   24107
```

files missing. I've applied the rest:

```
INFO:omero.util.populate_metadata:Created new table OriginalFile:6547687
INFO:omero.util.populate_metadata:Table initialized with 160 columns.
...
INFO:omero.util.populate_metadata:Created 208 MapAnnotations
```

This patch was needed:

```
[jamoore@minke experimentA]$ git diff idr0018-experimentA-bulkmap-config.yml
diff --git a/idr0018-neff-histopathology/experimentA/idr0018-experimentA-bulkmap-config.yml b/idr0018-neff-histopathology/experimentA/idr0018-experimentA-bulkmap-config.yml
index 08e0270..8a75ab3 100644
--- a/idr0018-neff-histopathology/experimentA/idr0018-experimentA-bulkmap-config.yml
+++ b/idr0018-neff-histopathology/experimentA/idr0018-experimentA-bulkmap-config.yml
@@ -187,7 +187,7 @@ columns:
     clientname: Phenotype 15 Term Accession URL
     clientvalue: http://www.ebi.ac.uk/cmpo/{{ value }}
     include: yes
-   - name: Phenotype 16
+  - name: Phenotype 16
     include: yes
   - name: Phenotype 16 Term Name
     include: yes
```

**Comment by joshmoore** [comment]
I'd guess this is caused by the fact that the ndpi reader appends a series number:

screen shot 2016-05-20 at 18 09 17

i.e. we'll have to decide which image we want as the searchable one.

**Comment by eleanorwilliams** [comment]
@joshmoore.  Extra space before Phenotype 16 removed.
Do you think it would work if I created a bulk annotation file with each of the 3 files in the .ndpi series named.  so that the annotations get added to each of them?

**Comment by joshmoore** [comment]
@eleanorwilliams : I'd imagine so. Let me know if you want me to test.

**Comment by eleanorwilliams** [comment]
@joshmoore  New annotations.csv file ready for testing.

**Comment by eleanorwilliams** [comment]
@joshmoore.  This one ready for re-loading with all three images in ndpi series listed in annotation file and extra columns added to bulkmap-config since table is not showing up in interface.

**Comment by joshmoore** [comment]
Had to remove this line from annotations.csv:

```
-Car4-13-922-brain,Car4-13_922-a54.tif,Mus musculus,brain,male,Car4-13-922,Car4 knockout,ENSMUSG00000000805,Car4,ENSG00000167434,CA4,RGB,derived,yes,,,,,,,,,,,,,,absence of cell spreading phenotype,absence of cell spre
```

**Comment by joshmoore** [comment]
Otherwise, :+1:

**Pull request merged**